tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
I guess resetGame() either doesn't run between the tests, or it does, but doesn't touch the cursor...?